src/SDKs/_metadata/cognitiveservices_data-plane_SpellCheck.txt

```diff
@@ -0,0 +1,11 @@
+2017-11-23 02:07:31 UTC
+
+1) azure-rest-api-specs repository information
+GitHub user: Azure
+Branch:      current
+Commit:      2e385816fdc99fbff09eb809e159eca464d8ec80
+
+2) AutoRest information
+Requested version: latest
+Bootstrapper version:    C:\Users\ashku\AppData\Roaming\npm `-- autorest@2.0.4204  
+Latest installed version:    
```

Comment:
Not sure  if I was supposed to edit this manually.  I was using a preview version of autorest to generate the spec since the codegen had a  bug.  Will perhaps try again once its released after thanksgiving and update this.